index.html

```diff
@@ -5,7 +5,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <link rel="stylesheet" href="style.css">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Microverse</title>
+    <title>Microverse.</title>
 </head>
 <body>
     <h1>Hellow Microverse</h1>
```

Comment:
![pr-title (copy)](https://user-images.githubusercontent.com/93039359/186142047-9c80dba5-11f8-46eb-b981-da86442f9fd0.png)


- [ ] Your branch name is not descriptive enough. You are using **scripts-addition**, you can use  **feature/setup-hello-microverse** or **feature/initialize-hello-microverse** :+1: